examples/html-ethers-5/src/main.js

```diff
@@ -0,0 +1,36 @@
+import { createWeb3Modal, defaultConfig } from '@web3modal/ethers-5'
+
+// @ts-expect-error 1. Get projectId
+const projectId = import.meta.env.VITE_PROJECT_ID
+if (!projectId) {
+  throw new Error('VITE_PROJECT_ID is not set')
+}
+
+// 2. Create wagmiConfig
+const chains = [1, 42161]
+
+// 3. Create modal
+const modal = createWeb3Modal({
+  ethersConfig: defaultConfig({
```

Comment:
Since these are examples, for clarity and consistency with wagmi, we should separate creation of `ethersConfig` into a separate step perhaps?